_posts/2020-03-05-bitcoin-abc-2020-fundraising.md

```diff
@@ -0,0 +1,39 @@
+
+Bitcoin ABC builds the most essential Bitcoin Cash infrastructure, our full node software which is used to produce almost every Bitcoin Cash block. Due to our past hard work on backports, maintenance, paying technical debt and more, Bitcoin ABC is now well-positioned to make rapid progress on the Bitcoin Cash scaling roadmap.
+
+The Bitcoin ABC project also performs the essential tasks of maintaining this software, providing support for it, backporting Bitcoin Core code, defending the Bitcoin Cash network from attacks and more.
+
+Currently, 90% of Bitcoin ABC developer time is absorbed with keeping Bitcoin Cash secure, running and updated. We must dedicate most of our minimal resources to “running in place” as we ensure the Bitcoin Cash network transcends scaling bottlenecks and remains up to date with the fast-moving Internet. Work on new features barely registers in the workload at this time precisely due to lack of funding. 
+
+But we are not content simply to preserve the status quo. Bitcoin ABC is building censorship-resistant P2P electronic cash for the masses - Bitcoin that scales.
+
+Bitcoin ABC is assembling a team of world-class engineering professionals to achieve this mission.
+
+Reality is constantly changing and we must change with it. Keeping up with the latest tools and technology requires funding. Advancing the Bitcoin Cash roadmap requires even more.
+
+The alternative is only to fall behind other cryptocurrency projects, to only worsen community polarization, to render increased adoption a hazardous endeavor and to only increase the eventual cost of building censorship-resistant P2P electronic cash for the world, assuming we are still able to at some later date.
+
+Bitcoin ABC has a strong funding plan that offers Bitcoin Cash miners, exchanges and other businesses the opportunity to sponsor Bitcoin ABC infrastructure development and get concrete benefits in return.
+
+We seek your feedback and participation in our sponsorship plan. We stand ready to hear your needs and see how Bitcoin ABC can help you achieve your goals.
+
+The goal of Bitcoin ABC is to build the software infrastructure needed for Bitcoin Cash to be successful. We are “building the roads” for Bitcoin Cash. We are enabling miners to achieve greater efficiency, and therefore greater profits.
+
+Join us in building censorship-resistant P2P electronic cash for the world and realizing the vision that is Bitcoin Cash!
+
+To schedule your presentation of the Bitcoin ABC 2020 Business Plan, please contact George Donnelly, Bitcoin ABC’s business development manager. Please note that priority for presentations is currently assigned to miners, exchanges and other Bitcoin Cash businesses.
+
+george@bitcoinabc.org
+[https://t.me/georgedonnelly](https://t.me/georgedonnelly)
```

Comment:
These last 3 lines need a blank line between them, or else they render on one line.

Reply:
Right, will see if I remember how to change that

Reply:
blank lines added